### PR TITLE
Added info about adding dependencies for demos

### PR DESCRIPTION
**Pull request by Blackbaud-DeniseCisneros**
Added to Getting Started section

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-update-demo-dependencies](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-update-demo-dependencies)

File: README.md [hunk 1]
```diff
@@ -98,6 +98,8 @@ export class AppExtrasModule { }
 </sky-docs-demo-page>
 ```
 
+- Add package dependencies for demos
```
Comment (Blackbaud-AlexKingman) [inline README.md:101]: Should we show an example here?
```
  <sky-docs-code-examples
    [packageDependencies]="{
      '@skyux/lookup': '*',
      'intl-tel-input': '*'
    }"
  >
```

Comment (Blackbaud-DeniseCisneros) [reply README.md:101]: Good question! I think we should

Comment (Blackbaud-DeniseCisneros) [reply README.md:101]: Done!

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-update-demo-dependencies](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-update-demo-dependencies)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-update-demo-dependencies](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-update-demo-dependencies)

**Pull request merged**